test-app/config/ember-try.js

```diff
@@ -41,30 +43,32 @@ module.exports = async function () {
           },
         },
       },
-      embroiderOptimized({
-        name: 'ember-lts-4.8 + embroider-optimized',
-        npm: {
-          devDependencies: {
-            'ember-source': '~4.8.0',
-            // @todo remove this once we have a stable release that includes https://github.com/embroider-build/embroider/pull/1383, which should be https://github.com/embroider-build/embroider/pull/1408
-            '@embroider/core': '2.1.2-unstable.c58f146',
-            '@embroider/compat': '2.1.2-unstable.c58f146',
-            '@embroider/webpack': '2.1.2-unstable.c58f146',
-          },
-        },
-      }),
-      embroiderOptimized({
-        name: 'ember-release + embroider-optimized',
-        npm: {
-          devDependencies: {
-            'ember-source': releaseVersion,
-            // @todo remove this once we have a stable release that includes https://github.com/embroider-build/embroider/pull/1383, which should be https://github.com/embroider-build/embroider/pull/1408
-            '@embroider/core': '2.1.2-unstable.c58f146',
-            '@embroider/compat': '2.1.2-unstable.c58f146',
-            '@embroider/webpack': '2.1.2-unstable.c58f146',
-          },
-        },
-      }),
+      // @todo Temporarily disabling embroider optimized tests due to https://github.com/CrowdStrike/ember-toucan-core/issues/210
```

Comment:
Nit: would have been enough to just not comment out the embroider scenarios in `ci.yml` as you did, and leave this as-is...

Reply:
Fixed with https://github.com/CrowdStrike/ember-toucan-core/pull/200/commits/0c8f95813e2019c0f55897b9ada00b61b6673916